pkg/registry/apis/secret/decrypt/client.go

```diff
@@ -27,19 +28,15 @@ func ProvideDecryptClientFunc(features featuremgmt.FeatureToggles, decryptStorag
 }
 
 // Decrypt a series of `names` from a `namespace`.
-func (d *DecryptClientFunc) Decrypt(ctx context.Context, namespace string, names ...string) (map[string]string, error) {
-	// TODO: ctx auth stuff.
+func (d *DecryptClientFunc) Decrypt(ctx context.Context, namespace string, names []string) (map[string]string, error) {
+	// TODO: Auth not needed because this is in process?
```

Comment:
you can get AuthInfo from context and verify it here -- no need to skip

Reply:
@ryantxu The question was more, do we even need this here since it's a function call in single tenant? What will we authorize against?